src/plugins/gameActivityToggle/index.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import "./style.css";
+
+import { getSettingStoreLazy } from "@api/SettingsStore";
+import ErrorBoundary from "@components/ErrorBoundary";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { Tooltip } from "@webpack/common";
+
+const ShowCurrentGame = getSettingStoreLazy<boolean>("status", "showCurrentGame");
+
+function GameActivityToggleButton() {
+    ShowCurrentGame?.useSetting();
+
+    return (
+        <Tooltip text="Toggle Game Activity">
+            {tooltipProps => (
+                <button
+                    {...tooltipProps}
+                    className="game-activity-toggle-btn"
+                    onClick={() => ShowCurrentGame?.updateSetting(!ShowCurrentGame.getSetting())}
```

Comment:
```suggestion
                    onClick={() => ShowCurrentGame?.updateSetting(s => !s)}
```
i think this is a thing? If so, please fix the type above